versioned_docs/version-2.0/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -72,3 +72,22 @@ To do so, select a dataset, click on "Examples", and then click the "Export Data
 This will open a modal where you can select the format you want to export to.
 
 ![Export Dataset Modal](../static/export-dataset-modal.png)
+
+## Create and manage dataset splits
+
+Dataset splits are divisions of your dataset which you can use to segment your data. For example, it is common
```

Comment:
```suggestion
Dataset splits are divisions of your dataset you can use to segment your data. For example, it is common
```

"which" is for non-restrictive clauses. Here, the predicate is restrictive so we should use "that" or no pronoun.